src/overlays/actors/ovl_En_Anubice/z_en_anubice.c

```diff
@@ -292,52 +299,52 @@ void EnAnubice_SetupDie(EnAnubice* this, GlobalContext* globalCtx) {
     this->animLastFrame = lastFrame;
     Animation_Change(&this->skelAnime, &gAnubiceFallDownAnim, 1.0f, 0.0f, lastFrame, ANIMMODE_ONCE, -20.0f);
 
-    this->unk_256 = false;
-    this->unk_258 = 0;
+    this->isFallingOver = false;
+    this->fallTargetPitch = 0;
     this->deathTimer = 20;
     this->actor.velocity.x = this->actor.velocity.z = 0.0f;
     this->actor.gravity = -1.0f;
 
     if (BgCheck_SphVsFirstPoly(&globalCtx->colCtx, &this->fireballPos, 70.0f)) {
-        this->unk_256 = true;
-        this->unk_258 = this->actor.shape.rot.x - 0x7F00;
+        this->isFallingOver = true;
+        this->fallTargetPitch = this->actor.shape.rot.x - 0x7F00;
```

Comment:
looking at this closer, im pretty sure that fireballPos is more correctly the headPos, looking at how it gets assigned in postLimbDraw.
it gets used to spawn the fireball initially, but its not the position of it as it travels. That makes this check here make alot more sense, if its head hits the wall (or any poly) as it falls, it tips over the other way to bounce off of it

Reply:
assuming im right (can maybe test in game to double check), this can be more named after `ricochet` or something of the effect